src/lib/env-vars.ts

```diff
@@ -153,6 +153,33 @@ export const DEVICE_TYPE_SPECIFIC_CONFIG_VAR_PROPERTIES: Array<{
 			},
 		},
 	},
+	{
+		capableDeviceTypes: [
+			'blackboard-tx2',
+			'jetson-tx2',
+			'n510-tx2',
+			'orbitty-tx2',
+			'skx2',
+			'spacely-tx2',
+			'srd3-tx2',
+		],
```

Comment:
```suggestion
		capableDeviceTypes: [
			'astro-tx2',
			'blackboard-tx2',
			'jetson-tx2',
			'n310-tx2',
			'n510-tx2',
			'orbitty-tx2',
			'spacely-tx2',
			'srd3-tx2',
		],
```

Updated list from @acostach.